transform/binary-plugin/binary_plugin.go

```diff
@@ -0,0 +1,78 @@
+package binary_plugin
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+
+	"github.com/konveyor/crane-lib/transform"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type BinaryPlugin struct {
+	commandRunner
+	log logrus.FieldLogger
+}
+
```

Comment:
Add a check to make sure that this conforms to: https://github.com/konveyor/crane-lib/blob/main/transform/plugin.go#L8

Reply:
+1

Reply:
I made the NewBinaryPlugin return `transform.Plugin` that implicitely makes the check. Is it necessary to add an explicit type check?